From f9e943b1be4179bfd33fc78e7fe90ffdb30aa64a Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Mon, 28 Sep 2015 18:01:27 +0800
Subject: [PATCH 0582/5242] MLK-11647 ARM: dts: imx6qp-sabresd: Correct DOVDD
 supply for mipi camera

commit  b3e3b27ab0e7824fb7e98f09f35d844a9030b335 from
https://source.codeaurora.org/external/imx/linux-imx.git

DOVDD ( GEN_1V8 ) is supplied by SW4 on i.MX6dqp-sabresd board
This patch corrects it.

Signed-off-by: Robby Cai <r63905@freescale.com>
(cherry picked from commit 50f1b6782d3ee8644129f98caff12c183411230c)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qp-sabresd.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabresd.dts b/arch/arm/boot/dts/imx6qp-sabresd.dts
index 59ce1e4..106645e 100644
--- a/arch/arm/boot/dts/imx6qp-sabresd.dts
+++ b/arch/arm/boot/dts/imx6qp-sabresd.dts
@@ -74,6 +74,10 @@
 	DOVDD-supply = <&sw4_reg>; /* 1.8v */
 };
 
+&ov564x_mipi {
+	DOVDD-supply = <&sw4_reg>; /* 1.8v */
+};
+
 &pcie {
 	power-on-gpio = <&gpio3 19 0>;
 	reset-gpio = <&gpio7 12 0>;
-- 
1.7.9.5

